roles/de-deploy-service/defaults/main.yml

```diff
@@ -4,3 +4,4 @@ deploy_use_color: false
 service_name_short: "{{service_name|replace('_', '-')}}"
 consul_port: 8500
 consul_token: "{% if consul is mapping and 'master_token' in consul %}{{ consul.master_token }}{% endif %}"
+data_container_name: "{{ data_container.container_name }}"
```

Comment:
This should probably be `data_container.compose_service`. Though they're the same string, what this is being used for below is for constructing a docker-compose service name, so we should use our names correctly.